rcl_action/src/rcl_action/goal_handle.c

```diff
@@ -68,6 +69,8 @@ rcl_action_goal_handle_init(
   goal_handle->impl->state = GOAL_STATE_ACCEPTED;
   // Copy the allocator
   goal_handle->impl->allocator = allocator;
+  // Set invaild time
+  goal_handle->impl->goal_terminal_timestamp = INT64_MAX;
```

Comment:
I think `INT64_MAX` can be still valid time? how about using `0` for initialized value instead?

```suggestion
  // Set invaild time
  goal_handle->impl->goal_terminal_timestamp = 0;
```

Reply:
Yes.